### PR TITLE
add support for specifying options when mounting filesystems

### PR DESCRIPTION
**Pull request by alexeyklyukin**
When mounting databases, we want to specify options like nobarrier or noatime to increase FS performance.

### CODE REVIEW

File: runtime/opt/zalando/init.d/10-prepare-disks.py [hunk 1]
```diff
@@ -113,13 +113,15 @@ def format_partition(partition, filesystem="ext4", initialize=False, is_already_
...
     """Mounts formatted disks provided by /etc/taupage.yaml"""
-    if is_mounted is False and dir_exists is False:
-        os.makedirs(mountpoint)
-        subprocess.check_call(["mount", partition, mountpoint])
-    elif is_mounted is False and dir_exists is True:
-        subprocess.check_call(["mount", partition, mountpoint])
+    if is_mounted is False:
+        if dir_exists is False:
+            os.makedirs(mountpoint)
+        if options and len(options) > 0:
```
Comment (hjacobs) [inline runtime/opt/zalando/init.d/10-prepare-disks.py]: you can leave "len(options) > 0" away, as options only is true if the list is non-empty

File: runtime/opt/zalando/init.d/10-prepare-disks.py [hunk 2]
```diff
@@ -113,13 +113,15 @@ def format_partition(partition, filesystem="ext4", initialize=False, is_already_
...
+            subprocess.check_call(["mount", "-o", options.replace(' ',''), partition, mountpoint])
```
Comment (hjacobs) [inline runtime/opt/zalando/init.d/10-prepare-disks.py]: would be good to stay DRY: only call mount once and prepare/add arguments when necessary

File: runtime/opt/zalando/init.d/10-prepare-disks.py (hunk 1 above)
Comment (alexeyklyukin) [inline runtime/opt/zalando/init.d/10-prepare-disks.py]: Removed, thank you!

File: runtime/opt/zalando/init.d/10-prepare-disks.py (hunk 2 above)
Comment (alexeyklyukin) [inline runtime/opt/zalando/init.d/10-prepare-disks.py]: Makes sense, changed the code to prepare the list of arguments and call mount only once.

**Comment by hjacobs** [comment]
Travis CI build fails: ./runtime/opt/zalando/init.d/10-prepare-disks.py:123:60: E231 missing whitespace after ','

(flake8)

**Comment by alexeyklyukin** [comment]
Doh, added that space :-)

**Pull request merged**